hack/jenkins/common.sh

```diff
@@ -71,7 +71,8 @@ echo ">> Downloading test inputs from ${MINIKUBE_LOCATION} ..."
 gsutil -qm cp \
   "gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-${OS_ARCH}" \
   "gs://minikube-builds/${MINIKUBE_LOCATION}/docker-machine-driver"-* \
-  "gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-${OS_ARCH}" out
+  "gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-${OS_ARCH}" out \
+  "gs://minikube-builds/${MINIKUBE_LOCATION}/gvisor-addon" out
```

Comment:
I think out needs to be specified at the end. Here you have the destination twice in this command.

Reply:
thanks for catching that, i'll fix this!